pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/clusterconfig/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: layout.pug
+navigationTitle: Cluster Configuration
+title: Cluster Configuration
+menuWeight: 20
+excerpt: Create a cluster configuration file and apply it to your environment.
+beta: false
+enterprise: false
+---
+
+## Create the cluster configuration file
+
+1.  Update cluster.sh with the existing infrastructure values.
+
+    ```sh
+    vi cluster-aws.sh
+    ```
+
+1.  Generate the cluster configuration file from the updated cluster.sh file.
+
+    ```sh
+    ./cluster.sh
+    ```
+
+    <p class="message--important"><strong>NOTE: </strong>AWS based AMI may contain a systemd process called autotune. Force autotune to allow ipv4_forward to be enabled in support of Kubernetes. Add the following to the <code>cluster-sbx.yaml</code>.</p>
+
+    ```yaml
+    kind: KubeadmControlPlane
+    ...
+        preKubeadmCommands:
+        - systemctl daemon-reload
+        - systemctl restart containerd
+        - /run/kubeadm/konvoy-set-kube-proxy-configuration.sh
+        - autotune override sysctl:net.ipv4.ip_forward:1
+        - systemctl restart autotune
+    ...
+    kind: KubeadmConfigTemplate
+          preKubeadmCommands:
+          - systemctl daemon-reload
+          - systemctl restart containerd
+          - /run/kubeadm/konvoy-set-kube-proxy-configuration.sh
+          - autotune override sysctl:net.ipv4.ip_forward:1
+          - systemctl restart autotune
+    ```
+
+    <p class="message--important"><strong>NOTE: </strong>Update the AMI root volume size to match new AMI generated with Konvoy Image Builder for both the control planes and default node pool.</p>
+
+    ```yaml
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+    kind: AWSMachineTemplate
+    metadata:
+      name: cluster-sbx-control-plane
+      namespace: default
+    spec:
+      template:
+        spec:
```

Comment:
I think this needs
```
cloudInit: 
  insecureSkipSecretsManager: false
```

Reply:
Getting this from https://mesosphere.slack.com/archives/C02MR1P7286/p1641503173008000?thread_ts=1641503115.007900&cid=C02MR1P7286

Reply:
yes. that is what we did to work around the issue. since its security related not sure of the total consequences of not using secrets manager. The secret is used to encrypt something with the start up process.